espGarden/main/include/wifi_manager.h

```diff
@@ -87,34 +36,71 @@ esp_err_t wifi_manager_get_ip(char* ip){
     return ESP_OK;
 }
```

Comment:
should we add this function call to the button press in heltec unofficial?

Reply:
You mean like getting the rssi on button press?
Maybe we should make a better display_print function that prints out the status of all the information on the display we care about. This would replace the display_centered_string. Any thoughts on what this should be printing?

Rssi?
IP?
Current status (WATERING ZONE X, Idle)?

Reply:
IP. RSSI can be removed unless we want to print it out on the display

Reply:
Updated.